rdrf/rdrf/templates/registration/login_assistance_sent.html

```diff
@@ -0,0 +1,15 @@
+{% extends "rdrf_cdes/base.html" %}
+{% load i18n %}
+
+{% block content %}
+<br>
+
+<div class="alert alert-info">
+{% trans "We've emailed you instructions for unlocking your account, if an account exists with the email you entered. You should receive them shortly." %}
```

Comment:
Can this become:

"If an account exists with the email you entered we will email you instructions to verify your account, . You should receive the email shortly."